src/measure_theory/bochner_integration.lean

```diff
@@ -554,16 +559,18 @@ variables [normed_space ℝ E] [nondiscrete_normed_field 𝕜] [normed_space 
 
 section integration_in_L1
 
-local notation `to_L1` := coe_to_L1 α E ℝ
-local attribute [instance] simple_func.normed_group simple_func.normed_space
+local notation `to_L1` := Lp.simple_func.coe_to_Lp α E ℝ
```

Comment:
`to_L1 := Lp.simple_func.coe_to_Lp α E ℝ`
Is that intentional? The L1 on the left and Lp on the right look strange.

Reply:
This notation is apparently never used!  I'll delete it.